modules/ztp-image-based-upgrade-installing-lifecycle-agent-using-web-console.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+// Epic TELCOSTRAT-160 (4.15/4.16), story TELCODOCS-1576
+// * scalability_and_performance/cnf-talm-for-cluster-upgrades.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-lifecycle-agent-using-web-console_{context}"]
+= Installing the {lcao} by using the web console
+
+You can use the {product-title} web console to install the {lcao} on both the seed and target cluster.
+
+.Prerequisites
+
+* Install the {rh-rhacm} Operator 2.9.2 or later.
```

Comment:
Why do we have ACM listed as a prerequisite for installing LCA using the web console?